recipe/meta.yaml

```
@@ -1,42 +1,57 @@
 {% set name = "bidict" %}
-{% set version = "0.21.2" %}
-{% set compress_type = "tar.gz" %}
-{% set hash_val = "4fa46f7ff96dc244abfc437383d987404ae861df797e2fd5b190e233c302be09" %}
+{% set version = "0.23.1" %}
+{% set commit_id = "636f5a610f0e0b9a1e056fc72b688720c114403b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ compress_type }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ hash_val }}
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 03069d763bc387bbd20e7d49914e75fc4132a41937fa3405417e1a5a2d006d71
+    # tests and tests data files
+  - url: https://github.com/jab/bidict/archive/{{ commit_id }}.tar.gz
+    sha256: 97c6b21db087960ed54752aaaf800cd52fe2f31cc3d48efde727c43e1704bf8d
+    folder: tests
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
-    - setuptools
-    - setuptools_scm
+    - setuptools >=40.9.0
+    - wheel
 
   run:
-    - python >=3.6
-    - setuptools
+    - python
 
 test:
   imports:
     - bidict
+  source_files:
+    - tests
```

If you did the change above, then the thing that you want to copy from the "sources" is the `gh_src` directory

```suggestion
    - gh_src
```

(Still another change coming!)